data/seed.js

```diff
@@ -3,26 +3,41 @@ const tickers = require('./tickers.js');
 const db = require('./index.js');
 
 // needs to create the array of all the tickers, and add the additional price data
-module.exports = {
-  start() {
-    let date = ['Q2 2018', 'Q3 2018', 'Q4 2018', 'Q1 2019', 'Q2 2019', 'Q3 2019', 'Q4 2019'];
-    let tickerList = tickers.createTickers().map(ticker => {
-      return {
-        ticker: ticker,
-        name: earnings.generateName(),
-        earnings: earnings.generateEarningsList()
-      };
-    });
-    return tickerList;
-  },
-
-  seedDatabase(cb = () => {}) {
-    // Creates a document in MongoDB for each ticker
-    db.Earnings.create(this.start(), (result) => {
-      console.log('Earnings seeded to database!');
-      cb();
-    });
+
+const start = () => {
+  let date = ['Q2 2018', 'Q3 2018', 'Q4 2018', 'Q1 2019', 'Q2 2019', 'Q3 2019', 'Q4 2019'];
+  let tickerList = tickers.createTickers().map(ticker => {
+    return {
+      ticker: ticker,
+      name: earnings.generateName(),
+      earnings: earnings.generateEarningsList()
+    };
+  });
+  return tickerList;
+}
+
+// able to save data by chunks
+// need to handle error
+async function seedDatabase() {
+  await db.Earnings.deleteMany({}, () => {
+    console.log('Remove Earnings')
+  })
+
+  let total = 50000
```

Comment:
what's this total referring to?

Reply:
This PR has an incorrect total. It should be 1000000